api/classes/prerequisites.js

```diff
@@ -153,4 +154,24 @@ prerequisites.validateCreationPermission = function(foreignKey, model) {
   };
 };
 
+/**
+ * trackTemporaryFile()
+ *
+ * Stores paths to temporary files in req.app for clearing after a request completes
+ * and in req.pre for use in the handler
+ */
+prerequisites.trackTemporaryFile = function() {
```

Comment:
Actually, I think this is 'file' now. At least ever since I figured out wtf was up with that two file nonsense :P